test/vanilla/Expected/AcceptanceTests/XmsErrorResponse/xmserrorresponse/operations/_pet_operations.py

```diff
@@ -58,9 +58,10 @@ def get_pet_by_id(
         """
         cls = kwargs.pop('cls', None)  # type: ClsType["models.Pet"]
         error_map = {
-            400: HttpResponseError,
-            404: lambda response: models.NotFoundErrorBaseException.from_response(response, self._deserialize),
-            501: HttpResponseError,
+            409: ResourceExistsError,
+            400: lambda response: HttpResponseError(response=response),
```

Comment:
That's unecessary complicated, since there is no model just writing the type should be enough (like you did for 409)

Reply:
Sounds good. I initially did this since it simplified the template, but will clean up so generated code is cleaner. thanks!